packages/@uppy/locales/script/build-en_US.mjs

```diff
@@ -6,7 +6,7 @@ import { fileURLToPath } from 'node:url'
 
 import { getLocales, sortObjectAlphabetically } from './helpers.mjs'
 
```

Comment:
```suggestion
// This script builds en_US by combining all keys from other languages and setting them to empty if they don't already exist
```
not sure how this script works, is it like that?

Reply:
No, we use `locale.ts` in all packages. They need to be combined into a single file (`en_US.ts`)